src/controllers/autoCompletion.controller.js

```diff
@@ -0,0 +1,25 @@
+const Query = require("../models/Query");
+
+exports.getAutoCompletions = async function (req, res, next) {
+  const histories = [];
+  const userInput = req.query.keywords;
+
+  try {
+    const matchingHistories = await Query.find(
+      { text: { $regex: `^${userInput}`, $options: "i" } },
+      {},
+    ).sort({ count: -1 });
+
+    matchingHistories
+      .slice(0, 5)
+      .forEach((element) => histories.push(element.text));
```

Comment:
숫자 5가 자동완성 검색어를 최대 5개까지 보낸주는 것 같습니다. 숫자 5보다는 상수로 처리하면 좀 더 직관적일 것 같아요!
```suggestion
    matchingHistories
      .slice(0, MAXIMUM_AUTO_COMPLETIONS)
      .forEach((element) => histories.push(element.text));
```

Reply:
네! 상수 처리하는게 가독성이 좋을 것 같네요! 반영해서 수정하겠습니다!